python/mxnet/profiler.py

```diff
@@ -54,5 +60,280 @@ def profiler_set_state(state='stop'):
 
 def dump_profile():
     """Dump profile and stop profiler. Use this to save profile
-    in advance in case your program cannot exit normally."""
+    in advance in case your program cannot exit normally.
+    """
     check_call(_LIB.MXDumpProfile())
+
+def dump_aggregate_stats(reset=False):
+    """Dump profile aggregate stats to console.
+    """
+    do_reset = 1 if reset is True else 0
+    check_call(_LIB.MXDumpAggregateProfileStats(int(do_reset)))
+
+def profiler_pause():
```

Comment:
remove `profiler_`. Its superfluous given mx.profiler.

We might also want to alias the existing apis to remove `profiler_`

Reply:
ok

Reply:
I don;t know what you're talking about with the "alias" comment -- what's that mean?  Remember I am not a python expert...